src/Farmer/Arm/Compute.fs

```diff
@@ -93,12 +94,9 @@ type VirtualMachine =
                 networkInterfaces.resourceId this.NetworkInterfaceName
                 yield! this.StorageAccount |> Option.mapList storageAccounts.resourceId
             ]
-            {| virtualMachines.Create(this.Name, this.Location, dependsOn, this.Tags) with
-                identity =
-                    if this.Identity = ManagedIdentity.Empty then Unchecked.defaultof<_>
-                    else this.Identity.ToArmJson
-                properties =
-                 {| hardwareProfile = {| vmSize = this.Size.ArmValue |}
+            let properties =
```

Comment:
Implemented this way because ARM chokes on any priority setting than 'Spot1 if `evictionPolicy` or `billingProfile` are included.